README.md

```diff
@@ -111,6 +111,46 @@ $ ./target/debug/fuel-core run --poa-instant=false
 2023-06-13T12:44:12.857763Z  INFO fuel_core::cli::run: 232: Block production disabled
 ```
 
+## Running a Mainnet node
+
+If you want to participate in the Mainnet network with your own node you can launch it following these simple commands.
+
+Install the latest fuelup installed :
+```bash
+curl -fsSL https://install.fuel.network/ | sh
+fuelup toolchain install latest
+```
+
+Clone the chain configuration folder :
+```
+git clone https://github.com/FuelLabs/chain-configuration
+```
+
+Generate a keypair for your node:
+```bash
+fuel-core-keygen new --key-type peering
+```
+and copy the secret key displayed.
+
+Run your mainnet node (change all variable with `{}` to your own personal variables):
+```bash
+fuel-core run \
+--enable-relayer \
+--service-name fuel-mainnet-node \
+--keypair {KEYGEN_SECRET_KEY} \
```

Comment:
perhaps these should be env vars, like

```
export KEYGEN_SECREY_KEY=<>
..etc
```

and then reuse in this line like this -

```
--keypair $KEYGEN_SECRET_KEY
```

Reply:
Mhhh it makes the "template command" more complicated to read and I think it's up to people on how they want to secure their credentials. What do you think on adding a sentence that says "instead of directly placing value in the command you can use, for example, environment variables."